sql/moz-fx-data-shared-prod/fenix_derived/feature_usage_events_v1/metadata.yaml

```diff
@@ -0,0 +1,21 @@
+friendly_name: Feature Usage Events
+description: |-
+  Metrics from events pings for mobile feature usage dashboards (Fenix)
+owners:
+- rzhao@mozilla.com
+labels:
```

Comment:
Can we add an application label here, just the way you did for `Feature Usage Metrics (Android - Fenix)`?